Src/VimCore/Modes_Insert_InsertMode.fs

```diff
@@ -329,6 +332,7 @@ type internal InsertMode
                 ("<C-g>", RawInsertCommand.CustomCommand this.ProcessUndoStart)
                 ("<C-^>", RawInsertCommand.CustomCommand this.ProcessToggleLanguage)
                 ("<C-k>", RawInsertCommand.CustomCommand this.ProcessDigraphStart)
+                ("<C-q>", RawInsertCommand.CustomCommand this.ProcessLiteralStart)

```

Comment:
Did not realize this was an alternate command for `control-v`.

Reply:
Given that 99% of the users who type `<C-v>` in insert mode in Visual Studio, will continue to expect it to work as paste from clipboard, I chose not to also assign that key. This is the old "do we default to Windows-friendly shortcuts on Windows or not." We could have a separate discussion about it. In the meantime, anyone who wants the pure vim `<C-v>` experience can map it to `<C-q>` in insert mode.